packages/ozone/src/mod-service/status.ts

```diff
@@ -126,14 +126,80 @@ const getSubjectStatusForModerationEvent = ({
   }
 }
 
+const recordEvents = [
+  'tools.ozone.moderation.defs#accountEvent',
+  'tools.ozone.moderation.defs#identityEvent',
+  'tools.ozone.moderation.defs#recordEvent',
+]
+
+const getSubjectStatusForRecordEvent = ({
+  event,
+  currentStatus,
+}: {
+  event: ModerationEventRow
+  currentStatus?: ModerationSubjectStatusRow
+}): Partial<ModerationSubjectStatusRow> => {
+  const timestamp = event.meta?.timestamp
+    ? `${event.meta?.timestamp}`
+    : event.createdAt
```

Comment:
The types here are a little funky to wrangle.  To avoid ending up with a string like `'true'` here, could we go with something more explicit like:
```suggestion
  const timestamp = typeof event.meta?.timestamp === 'string'
    ? event.meta.timestamp
    : event.createdAt
```